src/v2/ion/i18nTransformer.js

```diff
@@ -60,25 +60,29 @@ const I18N_OVERRIDE_MAPPINGS = {
 
   'identify-recovery.identifier': 'password.forgot.email.or.username.placeholder',
 
+  'select-authenticator-enroll.authenticator.duo': 'factor.duo',
```

Comment:
Are we just re-arranging entries here ?

Reply:
Yep - keeping them alphabetic so it is easier to follow!